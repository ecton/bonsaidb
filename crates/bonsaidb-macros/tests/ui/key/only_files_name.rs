use bonsaidb::files::FileConfig;

#[derive(FileConfig)]
#[file_config(files_name = "authority")]
struct Test;

fn main() {}
