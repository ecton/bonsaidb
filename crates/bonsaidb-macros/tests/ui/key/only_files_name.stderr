error: if `files_name` is specified, `blocks_name` needs to be provided as well
 --> tests/ui/key/only_files_name.rs:3:10
  |
3 | #[derive(FileConfig)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `FileConfig` (in Nightly builds, run with -Z macro-backtrace for more info)
